### ISSUE TITLE
Grammar Issue in PowerShellGet Version Warning

### ISSUE THREAD

**Issue by mirichmo** [PowerShell/PowerShellGetv2#250]
The warning message that a previous version of PowerShellGet is installed contains a small grammatical issue. "this command will install version '1.6.0' _**in side-by-side with**_ version '1.0.0.1'."

## Current Behavior
```powershell
Untrusted repository
You are installing the modules from an untrusted repository. If you trust this repository, change its
InstallationPolicy value by running the Set-PSRepository cmdlet. Are you sure you want to install the modules from
'PSGallery'?
[Y] Yes  [A] Yes to All  [N] No  [L] No to All  [S] Suspend  [?] Help (default is "N"): y
WARNING: Version '1.0.0.1' of module 'PowerShellGet' is already installed at 'C:\Program
Files\WindowsPowerShell\Modules\PowerShellGet\1.0.0.1'. To install version '1.6.0', run Install-Module and add the
-Force parameter, this command will install version '1.6.0' in side-by-side with version '1.0.0.1'.
PS C:\windows\system32> install-module powershellget -Force
```

## Possible Solution
Change wording to "side-by-side with."

## Steps to Reproduce (for bugs)
https://github.com/PowerShell/PowerShellGet/blob/115499efa248f33c9a349e3b210375346e8ccba2/PowerShellGet/PSGet.Resource.psd1#L33

## Your Environment

```PowerShell
Name                           Value
----                           -----
PSVersion                      5.1.16299.251
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.16299.251
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

**Comment by ghost** [comment]
To be clear, the proposal is to remove the word In. Yes, that should happen.

**Comment by bmanikm** [comment]
Resolved with #260.
